sunpy/map/maputils.py

```diff
@@ -101,7 +109,11 @@ def solar_angular_radius(coordinates):
     angle : `~astropy.units.Quantity`
         The solar angular radius.
     """
-    return sun._angular_radius(coordinates.rsun, coordinates.observer.radius)
+
+    if(_is_coordinates_frame_helioprojective(coordinates)):
```

Comment:
```suggestion
    if _is_coordinates_frame_helioprojective(coordinates):
```
You don't need these extra brackets.